Sage/carbon
# fix(action-popover): submenu is left aligned in the 'with submenu aligned right' story

### Proposed behaviour



Reduce the left margin in the Action Popover story [Kayboard Navigation Right Aligned Submenu](https://carbon.sage.com/?path=/story/action-popover--keyboard-navigation-right-aligned-submenu) so that the submenu correctly aligns to the right.
![2_09578656](https://github.com/Sage/carbon/assets/101639841/86115b46-4a8c-4e76-a918-76dd8dd9d234)

### Current behaviour


The submenu aligns to the left.
![1_309248](https://github.com/Sage/carbon/assets/101639841/3f58efcb-09f5-40be-9b31-edb987633678)


### Checklist




#### QA


### Additional context



### Testing instructions



- go to the storybook page.
- open the Action Popover storybook named: Keyboard Navigation Right Aligned Submenu.
- notice the submenu is left aligned.
